lib/app_gen/ecto_schema_reflector.ex

```diff
@@ -98,25 +98,28 @@ defmodule AppGen.EctoSchemaReflector do
     ArgumentError ->
       ecto_context = ecto_schema |> Module.split() |> Enum.drop(-1) |> Enum.join(".")
 
-      raise to_string(
-              IO.ANSI.format(
-                [
-                  :red,
-                  "Context module ",
-                  :bright,
-                  inspect(ecto_context),
-                  :reset,
-                  :red,
-                  " for schema ",
-                  :bright,
-                  inspect(ecto_schema),
-                  :reset,
-                  :red,
-                  " doesn't exist"
-                ],
-                true
-              )
-            )
+      reraise(
+        to_string(
+          IO.ANSI.format(
+            [
+              :red,
+              "Context module ",
+              :bright,
+              inspect(ecto_context),
+              :reset,
+              :red,
+              " for schema ",
+              :bright,
+              inspect(ecto_schema),
+              :reset,
+              :red,
+              " doesn't exist"
+            ],
+            true
+          )
+        ),
+        __STACKTRACE__
+      )
```

Comment:
Am I doing this reraise correctly?

Reply:
We actually don't need the stack trace in this error, since it's a library output

Reply:
That makes sense, I'll remove the stacktrace